### PR TITLE
Added some languages

### PR DESCRIPTION
**Pull request by tsaito18**

### CODE REVIEW

**Comment by googlefan256** [comment]
I think PowerShell might be same as shell script.
SIRANKEDO★

File: PowerShell/Hello-World.ps1 [hunk 1]
```diff
@@ -0,0 +1 @@
+echo "Hello, World!"
```
Comment (sevenc-nanashi) [inline PowerShell/Hello-World.ps1]:
```suggestion
Write-Output "Hello, World!"
```
I think it should use full name of command let

**Review by sevenc-nanashi — CHANGES_REQUESTED** [review]

File: Emojicode/Hello-World.emojic [hunk 2]
```diff
@@ -0,0 +1,3 @@
+🏁 🍇
+  😀 🔤Hello, Emojicode!🔤 ❗
```
Comment (waki285) [inline Emojicode/Hello-World.emojic]:
```suggestion
  😀 🔤Hello, World!🔤 ❗
```
It maybe Paru.

**Review by waki285 — CHANGES_REQUESTED** [review]

**Comment by nemupy** [comment]
Hello. Thanks for the pull request.
I have confirmed that all languages except Hodor work correctly.
Please wait a while longer.

**Comment by nemupy** [comment]
Confirmation of Hodor was made.
When executed, the output string was "Hello world!".
It is necessary to output "Hello, World!" instead of "Hello world!".
Thank you in advance for your help.

**Comment by tsaito18** [comment]
Thank you very much.
I have made the correction and would appreciate your confirmation.

**Comment by nemupy** [comment]
It's been verified to work correctly in all languages!
Thanks for the great request!

**Pull request merged**